src/get-started/java-binding.md

```diff
@@ -1371,9 +1371,47 @@ service sources and place it inside some known directory.
 Finally, add an `instances` section that describes the list of service instances
 you want to start in the blockchain. For each instance specify its
 artifact name alias in the `artifact` field. Instance names are the keys in the
-YAML dictionary. The Instance Configuration plugin also requires a list of
-additional parameters. In the present example the parameters refer to the
-`timestamping` instance:
+YAML dictionary.
+
+```yaml
+instances:
+  xnm-token:
+    artifact: cryptocurrency
+  time-oracle:
+    artifact: time
+```
+
+To instantiate a service which requires configuration parameters,
+`config` dictionary must be supplied.
+
+The Instance Configuration plugin supports several configuration formats:
+
+- Standard text formats: text, JSON, [java.util.Properties].
+- Arbitrary Protocol Buffers messages.
+
+#### Text Configuration Formats
+
+Requires the following parameters:
+
+- `format`. Describes the format of the configuration string. Possible values
+  are: `text`, `json` and `properties` (see [java.util.Properties]).
```

Comment:
shouldn't they be upper-cased i.e. TEXT?

Reply:
It would be inconsistent with the overall syntax of the configuration file. However, it's a good idea to allow it in plugin.